src/authorization.ts

```diff
@@ -20,44 +20,66 @@ export interface BulkDataAuth {
     importResources?: string[];
 }
 
+export type VerbType = 'CONNECT' | 'DELETE' | 'GET' | 'HEAD' | 'OPTIONS' | 'PATCH' | 'POST' | 'PUT' | 'TRACE';
+
+/**
+ * Contains information from incoming http request. As well as
+ * additional context information such use case specific auth handlers or

```

Comment:
Nitpick: I think we're missing an `as` in the comments.

"...information such `as` use case specific`

Reply:
Done.